Bottom: c17daefb8a46323dc21b7dca16319da53993b83e
Top:    04e5031e8764b2eb4a5f7bceb248d99ffe1caa01
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 17:38:44 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/lib/master.c b/lib/master.c
index 4bec1ef..175ecad 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1365,7 +1365,7 @@ static char *match_map_path(const char *match, const char *maps)
 	return 1;
 }
 
-static void write_map(struct source *source, struct mapent *first)
+static void write_map(struct map_source *source, struct mapent *first)
 {
 	struct mapent *me = first;
 	const char *map = source->argv[0];
